src/ResourceManager/Automation/Commands.Automation/Common/AutomationClientDSC.cs

```diff
@@ -1238,6 +1240,7 @@ public IEnumerable<Model.NodeConfiguration> ListNodeConfigurations(string resour
                     automationAccountName,
                     nodeConfigurationName,
                     null);
+
                 if (nodeConfigurationModel != null)
                 {
                     if (!overWrite)
```

Comment:
@cormacpayne - I could not understand the `ShouldContinue` part. Could you give a reference example to look at?